From af8cebe1a2d2d8315705cf42c558dee9773ec900 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v3-1.1-af8cebe1a2d-20220221T155556Z-avarab@gmail.com>
In-Reply-To: <cover-v3-0.1-00000000000-20220221T155556Z-avarab@gmail.com>
References: <cover-v2-0.2-00000000000-20220128T110330Z-avarab@gmail.com>
	<cover-v3-0.1-00000000000-20220221T155556Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 20 Dec 2021 13:35:23 +0100
Subject: [PATCH v3 1/1] Hotfix for merge of
 avar/add-new-sanitize-leak-test-modes and
 avar/support-test-verbose-under-prove-2

---
 t/t0000-basic.sh | 2 +-
 t/test-lib.sh    | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/t0000-basic.sh b/t/t0000-basic.sh
index 4f7657f5d0e..ffdfd931ca0 100755
--- a/t/t0000-basic.sh
+++ b/t/t0000-basic.sh
@@ -108,7 +108,7 @@ test_expect_success 'subtest: --immediate' '
 		<<-\EOF_OUT 3<<-EOF_ERR
 	> ok 1 - passing test #1
 	> not ok 2 - failing test #2
-	> #	false
+	> #false
 	> 1..2
 	EOF_OUT
 	EOF_ERR
diff --git a/t/test-lib.sh b/t/test-lib.sh
index 8803b93aafb..e20d55338ba 100644
--- a/t/test-lib.sh
+++ b/t/test-lib.sh
@@ -784,8 +784,8 @@ test_failure_ () {
 		# for "test_known_broken_ok_" and
 		# "test_known_broken_failure_", but work around it for
 		# now.
-		say_color error "not ok $test_count - # TODO pretending that 'not ok' was OK with --immediate --invert-exit-code ($1)"
-		say_color error "1..$test_count"
+		say_color_tap error "not ok $test_count - # TODO pretending that 'not ok' was OK with --immediate --invert-exit-code ($1)"
+		say_color_tap error "1..$test_count"
 		GIT_EXIT_OK=t
 		exit 0
 	fi
-- 
2.35.1.1132.ga1fe46f8690

